app/controllers/wikies_controller.rb

```diff
@@ -8,7 +8,7 @@ def search_results
   end
 
   def index
-    @allowed_collaborations = allowed_collaborations(current_user)
+    @allowed_collaborations = Collaboration.where(user_id: current_user.id).map(&:wiky)
```

Comment:
Hi @LuisMenjivar , take a look at the `has_many` associations declared in the `User` model. There's a simpler way to retrieve all of the `@allowed_collaborations` from `current_user`.

It's not clear if `@allowed_collaborations` ought to be wiki objects or collaboration objects. If `@allowed_collaborations` are meant to be wiki objects, consider renaming `@allowed_collaborations` to something else that is clearer about what the variable represents.